app/controllers/users_controller.rb

```diff
@@ -18,20 +18,19 @@ class UsersController < ApplicationController
   before_filter :return_here, :only => [:index, :show, :relationships, :dashboard, :curation]
   before_filter :before_edit, only: [:edit, :edit_after_auth]
   
-  MOBILIZED = [:show, :dashboard, :new, :create]
+  MOBILIZED = [:show, :new, :create]
   before_filter :unmobilized, :except => MOBILIZED
   before_filter :mobilized, :only => MOBILIZED
 
   protect_from_forgery unless: -> {
     request.parameters[:action] == "search" && request.format.json? }
 
-  caches_action :dashboard,
```

Comment:
Looks like `/dashboard` still queries the updates index for `user.recent_notifications` makes a few queries to the obs index to fetch observations via `get_local_onboarding_content` which will get called with every load. Is it OK if eventually that sidebar stuff is cached for just 15 minutes? Doesn't seem like a blocker for release, but I'd probably still prefer to cache `/dashboard` eventually